src/replication.c

```diff
@@ -1727,6 +1778,36 @@ int istgt_lu_create_snapshot(spec_t *spec, char *snapname, int io_wait_time, int
 	return (ret);
 }
 
+int
+istgt_lu_resize_volume(spec_t *spec, size_t size) {
+	replica_t *replica;
+	uint64_t io_seq;
+	zvol_op_code_t opcode = ZVOL_OPCODE_RESIZE;
+
+	MTX_LOCK(&spec->rq_mtx);
+
+	/* Wait for any ongoing snapshot commands */
+	while (spec->quiesce == 1) {
+		MTX_UNLOCK(&spec->rq_mtx);
+		sleep(1);
+		MTX_LOCK(&spec->rq_mtx);
+	}
+	io_seq = ++spec->io_seq;
+	ISTGT_LOG("Updating io_seq: %lu as part of resize operation\n", io_seq)
```

Comment:
instead of logging this, log a message which will log that resized the volume and also log the old size and new size.

Reply:
I will update log message